pom.xml

```diff
@@ -254,6 +263,11 @@
       <groupId>org.jetbrains.kotlin</groupId>
       <artifactId>kotlin-stdlib</artifactId>
     </dependency>
+    <dependency>
```

Comment:
I think, that this should be in the `pluginManagement` section or removed

Reply:
you're absolutely right about that. I removed this dependency from dependencies